strax/storage/mongo.py

```diff
@@ -202,8 +204,9 @@ def _close(self):
         # a TTL index -candidate
         if self.run_start is not None:
             update = {'run_start_time': self.run_start}
-            for chunk_id in self.ids_chunk.keys():
-                self.col.update_one({'_id': chunk_id}, {'$set': update})
+            query = {k: v for k, v in self.basic_md.items()
+                     if k in ('run_id', 'data_type', 'lineage_hash')}
```

Comment:
Ok, these looks like the fields to index. If `{**query}` above looks like this we can make a single compound index.

Reply:
Also we don't specify `run_id`, it looks like we use `number` instead.

Reply:
The query has already been indexed but not in a compound manner. I remember that compound indexing and TTLs don't go well together but these indexes are not part of the TTL indices.

Reply:
Mileage varies, obviously, but my usual strategy is to index for the specific queries I know are being made. If we always query against these three fields at the same time, a compound index probably provides a bit of a performance bonus. If we sometimes make queries against these fields individually, then a separate index might be better, or it might not.